### PR TITLE
[DRAFT] Install wagtail-localize

### PR DESCRIPTION
**Pull request by Pomax**
:warning: This PR should not be landed, it is a guiding PR for other PRs, instead :warning:

getting the changes from https://github.com/TheoChevalier/foundation.mozilla.org/commits/wagtail-2-11 into our localization dev branch.

partial fulfullment of https://github.com/mozilla/foundation.mozilla.org/issues/5999

### CODE REVIEW

File: network-api/networkapi/settings.py [hunk 1]
```diff
@@ -444,7 +465,7 @@
 # Rest Framework Settings
 REST_FRAMEWORK = {
     'DEFAULT_PERMISSION_CLASSES': [
-        'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly',
+        'rest_framework.permissions.AllowAny',
```
Comment (Pomax) [inline network-api/networkapi/settings.py:468]: this feels... dangerous. We don't want any random connection to have [full permissions without authenticating](https://www.django-rest-framework.org/api-guide/permissions/#allowany) @TheoChevalier?

Comment (TheoChevalier) [reply network-api/networkapi/settings.py:468]: That’s a good point… I’m really not familiar with this, but glancing at the doc it seems to be possible to set custom permissions?
I’ve added it as a task to the ticket

File: requirements.in [hunk 2]
```diff
@@ -9,21 +9,26 @@ django_csp
...
 django-storages
 factory_boy
 Faker
 filebrowser-safe
 future
 gunicorn
+pygit2==1.0.3
 python-slugify
+# Need to install a pre-release in order to get the RedLock.locked() method
+git+git://github.com/glasslion/redlock.git@master#egg=redlock
```
Comment (Pomax) [inline requirements.in:22]: This is not something we want to rely on, it's trivial for this to become malicious if there isn't a version or exact commit we can point to.

Comment (TheoChevalier) [reply requirements.in:22]: This is a backport of what we’re already using on Donate https://github.com/mozilla/donate-wagtail/blob/master/requirements.in#L20, so it’s something we will want to fix there too.

@kaedroho would you have an alternative to this? Maybe we don’t need a prerelease anymore?

Comment (KalobTaulien) [reply requirements.in:22]: Last commit to redlock was 2 years ago. If the pypi version doesn't work, let's absolutely link to the commit hash as a way to "lock" this package.

**Review by KalobTaulien — APPROVED** [review]
Let's make a note to add these env vars to production:
* WAGTAILLOCALIZE_PONTOON_GIT_CLONE_DIR
* WAGTAILLOCALIZE_PONTOON_GIT_URL

There's the requirement/dependecy github/master issue that needs resolving. Other than that, this looks good and works for me locally. I'm going to approve with the assumption the other issues get resolved.

File: requirements.in (hunk 2 above)
Comment (Pomax) [reply requirements.in:22]: If it's that old, I'd also worry about undocumented security vulnerabilities... is redlock our only option?

**Comment by Pomax** [comment]
Superceded by https://github.com/mozilla/foundation.mozilla.org/issues/6703

**Pull request closed without merging**